packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -108,9 +108,9 @@ async function runPipelineTransformStep(
         urlPath: `./${path.basename(rootFileName + destExt)}`,
       });
       if (typeof result === 'string') {
-        output[srcExt] = result;
+        output[destExt] = result;
```

Comment:
This is a bit difficult to follow, so here’s an example: `.vue` files will generate `.js` and `.css`. This transform function then runs twice for `.js` and then for `.css`.

What was happening was:

1. `output['.vue'] = [js transform]`
1. `output['.vue'] = [css transform]`

The original `srcExt` (`.vue`) was receiving all the transformations, then being blown away. This PR fixes that, and actually keeps the transformations.